specification/portalservices/resource-manager/Microsoft.PortalServices/extensions/readme.md

```diff
@@ -0,0 +1,53 @@
+# portal
```

Comment:
Is "extensions" also the folder name representing the customer-facing service name?